go/libkb/secret_store_other.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build !darwin,!android
+
+package libkb
+
+import "os"
+
+func NewSecretStoreAll(g *GlobalContext) SecretStoreAll {
+	// In order to not break production build releases, only
+	// use the SecretStoreFile on windows and linux if this
+	// environment variable is set.
+	if os.Getenv("KEYBASE_SECRET_STORE_FILE") != "1" {
+		return nil
+	}
+	return NewSecretStoreFile(g.Env.GetDataDir())
```

Comment:
On OSX this is the same as the config dir, but on Linux it isn't. Right now we keep secret keys in the config dir, which I think makes sense because they are Really Important Data That You Really Shouldn't Delete. (I think most of the other stuff in the Linux data dir is safe to nuke? I was never really clear on its intended meaning though.)